specification/containerservices/resource-manager/readme.md

```diff
@@ -0,0 +1,196 @@
+- Microsoft.ContainerService/2016-09-30/containerService.json
+```
+
+### Tag: package-2016-03
+
+These settings apply only when `--tag=package-2016-03` is specified on the command line.
+
+``` yaml $(tag) == 'package-2016-03'
+input-file:
+- Microsoft.ContainerService/2016-03-30/containerService.json
+```
+
+### Tag: package-2015-11-preview
+
+These setings apply only when `--tag=package-2015-11-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2015-11-preview'
+input-file:
+- Microsoft.ContainerService/2015-11-01-preview/containerService.json
+```
+
+### Tag: package-2015-06-preview
+
+These settings apply only when `--tag=package-2015-06-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2015-06-preview'
+input-file:
+- Microsoft.ContainerService/2015-11-01-preview/containerService.json
+```
+
+---
+
+# Code Generation
+
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  clear-output-folder: true
+```
+
+### Tag: package-2017-01 and go
+
+These settings apply only when `--tag=package-2017-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2017-01' && $(go)
+namespace: containerservice
+output-folder: $(go-sdk-folder)/services/compute/mgmt/2017-01-31/containerservice
+```
+
+### Tag: package-2016-09 and go
+
+These settings apply only when `--tag=package-2016-09 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2016-09' && $(go)
+namespace: containerservice
+output-folder: $(go-sdk-folder)/services/compute/mgmt/2016-09-30/containerservice
+```
+
+### Tag: package-2016-03 and go
+
+These settings apply only when `--tag=package-2016-03 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2016-03' && $(go)
+namespace: containerservice
+output-folder: $(go-sdk-folder)/services/compute/mgmt/2016-03-30/containerservice
+```
+
+### Tag: package-2015-11-preview and go
+
+These settings apply only when `--tag=package-2015-11-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2015-11-preview' && $(go)
+namespace: containerservice
+output-folder: $(go-sdk-folder)/services/compute/mgmt/2015-11-01-preview/containerservice
+```
+
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+
+``` yaml $(python)
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  namespace: azure.mgmt.containerservice
+  payload-flattening-threshold: 2
+```
+
+### Tag: package-2017-01 and python
```

Comment:
This tag "package-2017-01 and python" is not true anymore, if you could save me a PR and do it at the same time :)